docs.Dockerfile

```diff
@@ -21,5 +21,5 @@ COPY requirements.txt /mkdocs/
 WORKDIR /mkdocs
 VOLUME /mkdocs
 
-RUN apk --no-cache --no-progress add py3-regex py3-pip py3-wheel \
+RUN apk --no-cache --no-progress add gcc musl-dev python3-dev py3-regex py3-pip py3-wheel \
```

Comment:
Someone please review this. I have no idea, why I suddenly need `gcc` to build the docs. But otherwise
```
docker build -t doc-site:mkcos -f docs.Dockerfile .  
```
fails with `no such file or directory: gcc`

Reply:
I can run `docker build -t doc-site:mkcos -f docs.Dockerfile .` successfully locally without this change.

Docker version 20.10.6, build 370c289
MacOS Big Sur 11.1.

Reply:
I am running the same Docker version on my machine and it does work there without the change. If I remember correctly, the problem existed on the build server. I will try to revert the change and let GitHub build it again.

Reply:
Huh, it works now... I have no idea what else could have changed since I tried it last time.